src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -83,7 +83,7 @@ class ReportActionItemFragment extends React.PureComponent {
                     );
             case 'TEXT':
                 return (
-                    <Tooltip text={tooltipText} containerStyle={styles.w100}>
```

Comment:
Would it work for us to pass a prop here rather than putting a view wrapper an all of our tooltip components?

Reply:
Our `Tooltip` for the web already had a view wrapper, but our native version was just returning the children without any container (and consequently without the containerStyle). This PR is just adding a view wrapper to the native `Tooltip` too so we keep the styles consistent between native and web.

We use the `View` wrapper in the web version to [measure and position ](https://github.com/Expensify/Expensify.cash/blob/2f2f4c484ca42c7b1ec27c7df380cbe2e97e4a57/src/components/Tooltip/index.js#L84)the `Tooltip`, so removing it doesn't sound good.

Reply:
Sorry, let me rephrase that question. Can we accomplish this fix without making the switch to `flexShrink1`? The original issue just added `styles.w100`. What I'm asking is if we can keep that approach, but just make it something we can turn on and off with a prop. Does that make sense?

Reply:
Ah, I see! I don't think we can keep the `styles.w100` without causing the Tooltip to be misplaced, and flexShrink1 makes more sense here since it's specific to the issue at hand (i.e shrinking a component when it gets bigger than its flex container).

Would you prefer if the `styles.flexShrink1` was turned on by the `isSingleLine` prop? Something like
```
<Tooltip text={tooltipText} containerStyle={this.props.isSingleLine ? styles.flexShrink1 : undefined}>
```

Reply:
Ok, gotcha. I was trying to think of ways to avoid the wrapper, but this is definitely a clear way of doing it.